package.json

```diff
@@ -1,57 +1,61 @@
 {
   "name": "formidable-oss-landers",
-  "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
-    "formidable-oss-badges": "^0.3.5",
+  "version": "0.0.1",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "browser": "dist/formidable-oss-landers.umd.js",
+  "peerDependencies": {
+    "react": ">=16.8.0",
+    "styled-components": ">=5.1.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.10.5",
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.2.3",
+    "@babel/preset-react": "^7.10.4",
+    "@rollup/plugin-babel": "^5.2.0",
+    "@rollup/plugin-commonjs": "^15.0.0",
+    "@rollup/plugin-node-resolve": "^9.0.0",
+    "babel-eslint": "^10.1.0",
+    "builder": "^5.0.0",
+    "eslint": "^7.7.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.20.6",
+    "eslint-plugin-react-hooks": "^4.1.0",
+    "husky": "^4.2.5",
+    "prettier": "^2.0.5",
+    "prop-types": "^15.7.2",
     "react": "^16.13.1",
-    "react-copy-to-clipboard": "^5.0.2",
-    "react-dom": "^16.13.1",
-    "react-live": "^2.2.2",
-    "react-scripts": "3.4.1",
+    "rimraf": "^3.0.2",
+    "rollup": "^1.0.0",
+    "rollup-plugin-react-svg": "^3.0.3",
     "styled-components": "^5.1.1"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "storybook": "start-storybook -p 9009 -s public",
-    "build-storybook": "build-storybook -s public",
-    "format": "yarn run lint --fix",
-    "lint": "eslint 'src/**/*.js'"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
+    "build-babel": "babel src",
+    "build-umd": "rollup -c",
+    "build-babel-esm": "builder run build-babel -- -d es",
+    "build-babel-cjs": "builder run --env \"{\\\"BABEL_ENV\\\":\\\"cjs\\\"}\" build-babel -- -d lib",
+    "build": "builder concurrent --buffer build-babel-esm build-babel-cjs build-umd",
+    "clean": "rimraf dist lib es \"docs/dist\"",
+    "dev": "rollup -c -w",
+    "lint": "eslint '*.js' src 'docs/*.js' docs/src",
```

Comment:
We now have two linters working on `docs/` (`package.json:scripts.lint` and `docs/package.json:scripts/lint`). We should just have one in root or separate ones in root and docs.